### PR TITLE
update actix-web 3

### PR DESCRIPTION
**Pull request by KeenS**
actix-web 3 has been deleted and recommended to update to it

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/rust:1.39
+      - image: circleci/rust:1.46
```
Comment (KeenS) [inline .circleci/config.yml:5]: This is needed because some of dependencies fail to compile with Rust 1.39.0

**Comment by KeenS** [comment]
@pandaman64 please review :pray:

**Comment by pandaman64** [comment]
The repository is superseded by https://github.com/Idein/actix-extras due to project movement in the original Actix repo.

**Comment by KeenS** [comment]
I see. Why don't you archive this repo?

**Comment by pandaman64** [comment]
Just forgot this repo😅 Sorry for bothering you.
I'll look into updates in the other repo.

**Pull request closed without merging**